integrations/oci/sdk/cdi/pom.xml

```diff
@@ -37,25 +37,18 @@
         <spotbugs.exclude>etc/spotbugs/exclude.xml</spotbugs.exclude>
     </properties>
 
-    <dependencyManagement>
-      <dependencies>
-        <dependency>
-          <groupId>com.oracle.oci.sdk</groupId>
-          <artifactId>oci-java-sdk-shaded-full</artifactId>
-          <version>${version.lib.oci}</version>
-        </dependency>
-      </dependencies>
-    </dependencyManagement>
-
     <dependencies>
 
         <!-- Compile-scoped dependencies. -->
-
         <dependency>
             <groupId>com.oracle.oci.sdk</groupId>
-            <artifactId>oci-java-sdk-shaded-full</artifactId>
+            <artifactId>oci-java-sdk-common</artifactId>
             <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>com.oracle.oci.sdk</groupId>
+            <artifactId>oci-java-sdk-common-httpclient-jersey3</artifactId>
```

Comment:
(I'd add `<scope>compile</scope>` for clarity.)

Reply:
Updated as suggested.